R/ATTAINSCrosswalks.R

```diff
@@ -1723,3 +1760,561 @@ TADA_CreateUseParamRef <- function(.data, org_id = NULL, paramRef = NULL, usePar
   }
   return(CreateUseParamRef)
 }
+
+
+
+#' ATTAINS Assessment Unit and Use Name Crosswalk
+#'
+#' This function pulls in all prior ATTAINS Use names associated with each
+#' ATTAINS organization's Assessment Unit (AU) from the prior ATTAINS cycle.
+#' This function requires an ATTAINS org_id and 
+#' a crosswalk of an organization's WQP 
+#' Monitoring Location's, ATTAINS Assessment Unit's, and ATTAINS
+#' Water Type codes as a function input (sitesAURef). The output from 
+#' `TADA_GetATTAINS(.data, return_sf = FALSE)` can be used 
+#' directly as the sitesAURef argument input in this function. Alternatively, 
+#' a user supplied crosswalk can be entered or `TADA_GetATTAINSAUSiteCrosswalk()`
+#' and/or `TADA_UpdateMonitoringLocationsInATTAINS()` functions can be leveraged 
+#' to generate the crosswalk.
+#'
+#' This function is mainly designed to assist with pulling
+#' existing Uses that have been entered
+#' into ATTAINS in the prior ATTAINS cycle (most recent assessment).
+#' 
+#' For any NEW AUs and/or NEW uses, users must modify 
+#' the output of this function to manually add those uses and AU's to the crosswalk.
+#' Alternatively, we have developed a helper function, [TADA_CreateWaterUseRef()], 
+#' to assist with assigning uses to NEW AU's. This can be levereaged to assign
+#' uses for any new AUs based on the water type of the AU.
+#' Users can either supply their own Water
+#' Type to Use crosswalk or utilize ATTAINS webservices to pull in the Water Type to
+#' Use reference file. This Water to Use reference file can be used to assign all
+#' unique Uses to a new/modified AU based on which uses have been assisgned to that
+#' water type in the past for the specified ATTAINS organization. 
+#' Any new or modified AU and use information that gets submitted to ATTAINS 
+#' in the current assessment cycle will not be available in ATTAINS until the
+#' assessment is approved and completed. 
+#' 
+#' @param .data A TADA dataframe. The user should run all desired data cleaning,
```

Comment:
How is .data used in this function? Will this function will look for WQP Monitoring 
Location Identifier's and  ATTAINS Assessment Units in .data if they are there from running TADA_GetATTAINS first? Or is sitesAURef the source of the site/AU associations? Should TADA_GetATTAINS  include a sitesAURef output to streamline this?